client/src/components/Header/components/universal-nav.css

```diff
@@ -198,6 +199,25 @@ button.nav-link[aria-disabled='true'] {
   margin-bottom: 0;
 }
 
+.exposed-nav-list {
+  list-style-type: none;
+  min-height: calc((var(--header-height) + 2.5px));
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  padding: 0 !important;
+}
+
+.exposed-nav-elements {
+  margin-right: 3px;
+  padding: 7.5px 3px;
+  padding-bottom: 8px;
+}
```

Comment:
we can have the same styles as the menu, then if we need improvement, we can improve both menu and the new button.

```suggestion
.exposed-nav-list {
  list-style-type: none;
  min-height: calc((var(--header-height) + 2.5px));
  display: flex;
  align-items: center;
  gap: .25em;
  flex-direction: row;
  padding: 0;
}

.exposed-nav-elements {
  margin-right: 3px;
  max-height: calc(var(--header-height) - 8px);
  display: flex;
  align-items: center;
  padding: 2px 14px;
  background-color: var(--theme-color);
  color: var(--gray-00);
  border: 1px solid var(--gray-00);
}
```

Reply:
the new looks 
![image](https://user-images.githubusercontent.com/88248797/203540867-874f709c-0e78-4fb2-9506-2071d7c80014.png)

the old looks
![image](https://user-images.githubusercontent.com/88248797/203541201-326caea0-ce57-4af3-b9ef-a2b6b46bd231.png)

Reply:
I don't know, the menu button is different from forum, news and radio. All these buttons take you to a different page, while the menu button shows you options on the same page. Their characteristics are different and thus I think they should look different. But if you insist I am okay if they look alike. What do you think?

Reply:
True good point 👍